pkg/cli/debug/debug.go

```diff
@@ -968,6 +968,10 @@ func (o *DebugOptions) approximatePodTemplateForObject(object runtime.Object) (*
 			// TODO: allow --as-root=false to skip all the namespaces except network
 			return nil, fmt.Errorf("can't debug nodes without running as the root user")
 		}
+		if t.Labels["kubernetes.io/os"] == "windows" {
+			// Windows nodes don't yet support privileged containers, which is required for debug
```

Comment:
Why do you need priviledged containers for debugging a pod? Can't you run `cmd` like you were doing in #907?

Reply:
It doesn't work, and the docs explained that debug needs privileged containers, and that Windows nodes don't currently support that: https://docs.openshift.com/container-platform/4.9/support/troubleshooting/troubleshooting-windows-container-workload-issues.html#accessing-windows-node

This PR is to address a bug where running `oc debug node` currently hangs attempting to debug a Windows node. If we can support debugging Windows nodes, that would be great though.

Reply:
That is correct. `oc debug node` does not work for Windows nodes. The main confusion here is the association with [Bug 1990014  - oc debug <pod-name> does not work for Windows pods](https://bugzilla.redhat.com/show_bug.cgi?id=1990014) which has to be debugging Windows Pods. I want to ensure that we are not disabling that.

Reply:
Correct, in that BZ, someone tried both nodes and pods. The intent is to provide users with better feedback when `oc debug node` is attempted on Windows nodes. Once this is merged, I plan to set the bug back to NEW to address the issue with pods, as that is a valid bug.

Reply:
@deejross, Thank you for the detailed explanation.